sanket/javascript/assignment10(day28)/sanketA10_Q8.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>alpha numeric dash and underscore</title>
+  </head>
+  <body>
+    <script>
+      function check(value) {
+        let regExp = /[\w-]/;
```

Comment:
not working as expected , even if there is no _ or - still it says the string has underscorre and dash